### PR TITLE
docs: Fix incorrect and outdated saucectl cli content

### PR DESCRIPTION
**Pull request by tianfeng92**
Fix incorrect and outdated saucectl cli content

### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/dev/cli/saucectl/run.md [hunk 1]
```diff
@@ -412,10 +483,80 @@ saucectl run --region us-west-1
...
+```
+
+</div>
+
+### <span className="cli">--reporters.json.webhookURL</span>
+
+<div className="cli-desc">
+<p><small>| OPTIONAL | URL |</small></p>
+
+Specifies the webhook URL. When saucectl test is finished, it'll send a HTTP POST payload to the configured webhook URL.
```
Comment (alexplischke) [inline docs/dev/cli/saucectl/run.md:519]:
```suggestion
Specifies the webhook URL. When saucectl test is finished, it'll send an HTTP POST payload to the configured webhook URL.
```

File: docs/dev/cli/saucectl/usage/ci/github-actions.md [hunk 2]
```diff
@@ -59,9 +64,20 @@ For more detailed information on setting event-driven actions and jobs, visit th
...
+```yaml
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Saucectl RUN
+        uses: saucelabs/saucectl-run-action@v3
```
Comment (alexplischke) [inline docs/dev/cli/saucectl/usage/ci/github-actions.md:76]: Might as well use the latest available one:
```suggestion
        uses: saucelabs/saucectl-run-action@v4
```

File: docs/dev/cli/saucectl/usage/ci/github-actions.md [hunk 3]
```diff
@@ -59,9 +64,20 @@ For more detailed information on setting event-driven actions and jobs, visit th
 :::
 
 
-```yaml reference
```
Comment (alexplischke) [inline docs/dev/cli/saucectl/usage/ci/github-actions.md:62]: Curious as to why you didn't want to keep the reference here?

Comment (tianfeng92) [reply docs/dev/cli/saucectl/usage/ci/github-actions.md:62]: The reference currently is not accurate due to content updates.

For instance, the existing YAML reference at:
https://docs.saucelabs.com/dev/cli/saucectl/usage/ci/github-actions/#create-the-test-job
is shown below:
Screenshot displaying a partial view of the YAML configuration

This does not comprehensively cover the setup steps for the entire job.

While referencing a specific commit could ensure accuracy, it also runs the risk of becoming outdated over time.

Therefore, I chose to directly include the YAML configuration here and provide a link to an example in the GitHub repository for a more reliable and up-to-date reference.

Comment (tianfeng92) [reply docs/dev/cli/saucectl/usage/ci/github-actions.md:62]: I agree. My update here ensures there's at least a link to content that is both accurate and up-to-date.

**Review by alexplischke — APPROVED** [review]

**Pull request merged**